test/expect/TestJit.test_fusion_distribute.expect

```diff
@@ -1,20 +1,16 @@
 graph(%0 : Float(4, 4)
       %1 : Float(4, 4)) {
-  %2 : int = prim::Constant[value=1]()
-  %3 : int = prim::Constant[value=2]()
-  %4 : Float(4!, 2), %5 : Float(4!, 2) = aten::chunk(%0, %3, %2)
-  %6 : Float(4!, 2), %7 : Float(4!, 2) = aten::chunk(%1, %3, %2)
-  %8 : Float(4, 2) = prim::FusionGroup_0[device=0](%4, %6, %5, %7)
-  return (%8);
+  %2 : Float(4, 2) = prim::FusionGroup_0[device=0](%0, %1)
```

Comment:
This test is for distributing chunk over pointwise ops we could fuse. Is that transformation still a thing? We should be able to fuse all chunks anyway, right? Did you remove it?

Reply:
We still distribute the chunks before we fuse any chunks in. Fusing the chunks in first prevents possible fusion opportunities (because the invariant for prim::FusedChunk is that only inputs to the FusionGroup can be inputs to prim::FusedChunk).

This expect files shows that the chunk was distributed over the fusible pointwise ops, then everything was fused into one fusion group